client/src/app/site/base/list-view-base.ts

```diff
@@ -144,6 +167,34 @@ export abstract class ListViewBaseComponent<V extends BaseViewModel, M extends B
         this.dataSource.filter = event;
     }
 
+    /**
+     * Initialize the settings for the paginator in every list view.
+     */
+    private async initializePagination(): Promise<void> {
+        // If the storage is not available - like in history mode - do nothing.
+        if (this.storage) {
+            this.storageObject = (await this.storage.get('Pagination')) || {};
+            // Set the number of items per page -- by default to 25.
+            this.paginator.pageSize = this.storageObject[this.storageKey] || 25;
+            // Subscription to page change events, like size, index.
+            this.subscriptions.push(
+                this.paginator.page.subscribe(async (event: PageEvent) => {
+                    await this.setPageSettings(event.pageSize);
+                })
+            );
+        }
+    }
+
+    /**
+     * Function to set the new selected page size in the browser's local storage.
+     *
+     * @param size is the new page size.
+     */
+    public async setPageSettings(size: number): Promise<void> {
```

Comment:
If the `pagiantionStorageObject` does not exist, either cancel this method or query it explicitly.